examples/hello/package.json

```diff
@@ -5,11 +5,6 @@
   "types": "index.d.ts",
```

Comment:
didn't we say we wanted package.json files to be generated?

Reply:
I kept this and the other example package.json files in case you wanted them available for users to take as examples as well. I agree it doesn't make sense project-wise to keep them, and we don't actually use them in any real way here. I can get rid of them if you want, but I didn't want to delete them unless you were sure.

Reply:
Let's get rid of them.